src/components/ActionToolbar.vue

```diff
@@ -7,10 +7,15 @@
       </button>
 
       <action-toolbar-button
-        v-for="button in buttons"
+        v-for="(button, index) in formattedButtons"
         :icon="button.icon"
         :label="button.label"
         :key="button.icon"
+        :category="button.category"
+        :is-active="button.isActionToolbarMenuOpened"
+        :class="{ 'action-toolbar__btn--active': button.isActionToolbarMenuOpened, 'action-toolbar__btn--disable': !button.enable }"
```

Comment:
A good practice is to used a computed here in order to be concise and keep conventions from Tucana